contracts/interfaces/IRoninGovernanceAdmin.sol

```diff
@@ -18,6 +18,8 @@ interface IRoninGovernanceAdmin {
   event EmergencyExitPollApproved(bytes32 _voteHash);
   /// @dev Emitted when an emergency exit poll is expired.
   event EmergencyExitPollExpired(bytes32 _voteHash);
+  /// @dev Emitted when an emergency exit poll is voted.
+  event EmergencyExitPollVoted(bytes32 indexed _voteHash);
```

Comment:
I think we should add the address of voter in the event, as in [event ProposalVoted(bytes32 indexed proposalHash, address indexed voter, Ballot.VoteType support, uint256 weight);](https://github.com/axieinfinity/ronin-dpos-contracts/blob/63f1b56707e2ef5c093d2ea7eaee583eb77bdca7/contracts/extensions/sequential-governance/CoreGovernance.sol#L46)

```suggestion
  event EmergencyExitPollVoted(bytes32 indexed proposalHash, address indexed voter);
```